### PR TITLE
Add dynamic warning for product saving

### LINKED ISSUE #455: Na het aanpassen van de tekst is dit niet duidelijk terug te zien en is ook niet meer duidelijk hoe je het moet opslaan.
When someone edits a text, please show a small yellow information bar above the field that says you still need to save the changes:

"U dient uw wijzigingen nog op te slaan als concept of te publiceren, onderaan deze pagina."

### PR DESCRIPTION
**Pull request by dariomory**
Fixes #455

_______

**Changes**

- Added dynamic warning for product saving

Peek 2022-03-08 16-47

### CODE REVIEW

File: src/sdg/scss/components/_tabs.scss [hunk 1]
```diff
@@ -100,6 +100,32 @@
     line-height: 1.5;
     color: #212121;
 
+    &--warning {
```
Comment (svenvandescheur) [inline src/sdg/scss/components/_tabs.scss:103]: I don't necessarily like the way this is set up. Is, this a tab component, a table component, or a notification component?

The (BEM) methodology we use indentifies 3 layers of naming.

Block - A standalone, reusable component). (e.g. `.tabs`).
Element - A dependency of a block, without any meaning on it's own (e.g. `.tabs__tab`).
Modifier A theme or a state of either a block or an element (e.g. `.tabs__tab--active).

__Please not that a modifier should always be combined with the block or element classname__

So do:

```html
<div class="tabs__tab tabs__tab--active"></div>
```

And dont do:

```html
<div class="tabs__tab--active"></div>
```

The idea of BEM is that everything is as scoped (isolated) and reusable as possible and is able to express some minimal state using simple class names. 

In this case, `tabs` is a good example of a BEM block, however  `tabs__table` is not a good example of an element. A better solution would be to create a new file `table` which contains the styling for a table without any dependency on tabs (as we would like to be able to use a table outside the context of a tab). A better HTML tree would look something like:

```html
<div class="tabs">
  (...)
  <div class="tabs_tab tabs__tab--active">
    <table class="table>
      (...)
    </table>
  </div>
</div>
```

If a tab would override de styling of a table, a better solution would be something like `.tabs .table`.

File: src/sdg/scss/components/_tabs.scss [hunk 2]
```diff
@@ -100,6 +100,32 @@
...
+      border: 1px solid darken($color_accent_notification-bg, 10%);
+      color: $color_accent_darkest;
+      height: 22px;
+      font-size: 10px;
+      padding: 4px 8px;
+      text-align: end;
+      position: relative;
+      top: -1px;
+      filter: opacity(0.8);
+      transition: all 0.3s ease-in-out;
```
Comment (svenvandescheur) [inline src/sdg/scss/components/_tabs.scss:120]: Try to keep your formatting in order, it's tyically better not to nest to deep and have newlines between selectors. Bonus points for sorting properties alphatibetically.

**Review by svenvandescheur — CHANGES_REQUESTED** [review]
Lets discuss this together.

**Comment by dariomory** [comment]
Discussed. Fixed in #478

**Pull request closed without merging**